src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big">
```

Comment:
In HTML5, the `<form>` tag should not be self-closed. It should be written as `<form>` and closed with `</form>`.